contents/docs/integrate/_snippets/install-flutter.mdx

```diff
@@ -6,63 +6,116 @@ Set your PostHog API key and change the automatic event tracking on if you wish
+```
+
+#### iOS setup
+
+For iOS, you'll need to have [Cocoapods](https://guides.cocoapods.org/using/getting-started.html) installed. Then add your PostHog configuration to the `Info.plist file` located in the `ios/Runner` directory:
+
+```xml ios/Runner/Info.plist
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	[...]
-	<key>com.posthog.posthog.API_KEY</key>
-	<string><ph_project_api_key></string>
-	<key>com.posthog.posthog.POSTHOG_HOST</key>
-	<!-- usually 'https://app.posthog.com' or 'https://eu.posthog.com' -->
-	<string><ph_instance_address></string>
-	<key>com.posthog.posthog.CAPTURE_APPLICATION_LIFECYCLE_EVENTS</key>
-	<true/>
-	<key>com.posthog.posthog.DEBUG</key>
-	<true/>
-	[...]
+<!-- rest of your configuration -->
+  <key>com.posthog.posthog.API_KEY</key>
+  <string><ph_project_api_key></string>
+  <key>com.posthog.posthog.POSTHOG_HOST</key>
+  <string><ph_instance_address></string>
+  <key>com.posthog.posthog.CAPTURE_APPLICATION_LIFECYCLE_EVENTS</key>
+  <true/>
 </dict>
 </plist>
 ```
 
-#### Web
+Then you need to set the minimum platform version to iOS 13.0 in your Podfile:
+
+```yaml ios/Podfile
+platform :ios, '13.0'
+
+# rest of your config
+```
+
+#### Web setup
 
-```html file=index.html
+For Web, add your `Web snippet` (which you can find in [your project settings](https://us.posthog.com/settings/project#snippet)) in the `<body>` of your `web/index.html` file:
+
+```html file=web/index.html
 <!DOCTYPE html>
 <html>
-  <head>
-    ...
-    <script>
-      !function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.async=!0,p.src=s.api_host+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="capture identify alias people.set people.set_once set_config register register_once unregister opt_out_capturing has_opted_out_capturing opt_in_capturing reset isFeatureEnabled onFeatureFlags getFeatureFlag getFeatureFlagPayload reloadFeatureFlags group updateEarlyAccessFeatureEnrollment getEarlyAccessFeatures getActiveMatchingSurveys getSurveys".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
-      <!-- usually 'https://app.posthog.com' or 'https://eu.posthog.com' -->
-      posthog.init('<ph_project_api_key>', {api_host: '<ph_instance_address>'})
-    </script>
-  </head>
-
-  <body>
-    ...
-  </body>
+
+<head>
+  <!-- ... other head elements ... -->
+</head>
```

Comment:
This is different from https://posthog.com/docs/libraries/flutter#web
the script goes within the head, maybe even the script is different, not sure.